modules/virt-creating-configmap.adoc

```diff
@@ -0,0 +1,264 @@
+    openshift.io/support-url: https://github.com/kubevirt/common-templates/issues
+    tags: hidden,kubevirt,virtualmachine,windows
+    template.kubevirt.io/editable: |
+      /objects[0].spec.template.spec.domain.cpu.cores
+      /objects[0].spec.template.spec.domain.resources.requests.memory
+      /objects[0].spec.template.spec.domain.devices.disks
+      /objects[0].spec.template.spec.volumes
+      /objects[0].spec.template.spec.networks
+    template.kubevirt.io/version: v1alpha1
+    template.openshift.io/bindable: "false"
+    validations: |
+      [
+        {
+          "name": "minimal-required-memory",
+          "path": "jsonpath::.spec.domain.resources.requests.memory",
+          "rule": "integer",
+          "message": "This VM requires more memory.",
+          "min": 536870912
+        }
+      ]
+  labels:
+    flavor.template.kubevirt.io/medium: "true"
+    os.template.kubevirt.io/windows: "true"
+    template.kubevirt.io/type: base
+    template.kubevirt.io/version: v0.11.2
+    workload.template.kubevirt.io/server: "true"
+  name: custom-windows-server-medium-v0.7.0
+  namespace: openshift
+objects:
+- apiVersion: kubevirt.io/v1alpha3
+  kind: VirtualMachine
+  metadata:
+    labels:
+      app: ${NAME}
+      vm.kubevirt.io/template: custom-windows-server-medium-v0.7.0
+      vm.kubevirt.io/template.revision: "1"
+      vm.kubevirt.io/template.version: v0.11.2
+    name: ${NAME}
+  spec:
+    running: false
+    template:
+      metadata:
+        labels:
+          kubevirt.io/domain: ${NAME}
+          kubevirt.io/size: medium
+      spec:
+        domain:
+          clock:
+            timer:
+              hpet:
+                present: false
+              hyperv: {}
+              pit:
+                tickPolicy: delay
+              rtc:
+                tickPolicy: catchup
+            utc: {}
+          cpu:
+            cores: 1
+            sockets: 1
+            threads: 1
+          devices:
+            disks:
+            - disk:
+                bus: sata
+              name: rootdisk
+            interfaces:
+            - masquerade: {}
+              model: e1000e
+              name: default
+          features:
+            acpi: {}
+            apic: {}
+            hyperv:
+              relaxed: {}
+              spinlocks:
+                spinlocks: 8191
+              vapic: {}
+          machine:
+            type: pc-q35-rhel8.2.0
+          resources:
+            requests:
+              memory: 4Gi
+        evictionStrategy: LiveMigrate
+        networks:
+        - name: default
+          pod: {}
+        terminationGracePeriodSeconds: 0
+        volumes:
+        - name: rootdisk
+          persistentVolumeClaim:
+            claimName: ${PVCNAME}
+parameters:
+- description: VM name
+  from: windows-[a-z0-9]{6}
+  generate: expression
+  name: NAME
+- description: Name of the PVC with the disk image
+  name: PVCNAME
+  required: true
+EOF
+----
+
+. Create a custom ConfigMap:
```

Comment:
I changed this to "Create a custom ConfigMap based on the following example:" The user should not be copying and running the entire command because only one operating system mapping will be relevant. So the custom ConfigMap is an example rather than a command.